devops_azure.ps1

```diff
@@ -0,0 +1,39 @@
+# Description: Boxstarter Script
+# Author: Microsoft
+# Common settings for azure devops
+
+Disable-UAC
+
+# Get the base URI path from the ScriptToCall value
+$bstrappackage = "-bootstrapPackage"
+$helperUri = $Boxstarter['ScriptToCall']
+$strpos = $helperUri.IndexOf($bstrappackage)
+$helperUri = $helperUri.Substring($strpos + $bstrappackage.Length)
+$helperUri = $helperUri.TrimStart("'", " ")
+$helperUri = $helperUri.TrimEnd("'", " ")
+$helperUri = $helperUri.Substring(0, $helperUri.LastIndexOf("/"))
+$helperUri += "/scripts"
+write-host "helper script base URI is $helperUri"
+
+function executeScript {
+    Param ([string]$script)
+    write-host "executing $helperUri/$script ..."
+	iex ((new-object net.webclient).DownloadString("$helperUri/$script"))
+}
+
+#--- Setting up Windows ---
+executeScript "FileExplorerSettings.ps1";
+executeScript "SystemConfiguration.ps1";
+executeScript "RemoveDefaultApps.ps1";
+executeScript "WSL.ps1";
+
+# TODO: Expand on tools/configuration options here
+
+
+#--- Browsers ---
+choco install -y googlechrome
```

Comment:
maybe have this be "Browsers.ps1"

Reply:
this would include FireFox as well.

Reply:
hmm.... yeah... I didn't intend this script to be done.  I was putting it in here intending to work on it more after this PR.  Wanted to get the demos folder added (Michel is planning on also adding a demo script soon).

Reply:
I think the web tooling one has this as well. I think we have a bug for that